### PR TITLE
conda: fix deactivate example

### PR DESCRIPTION
**Pull request by andrewmarkallen**
- [ ] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

`conda deactivate environment` isn't a valid command. While calling deactivate without an environment based on the tldr example might seem obvious to experienced coders, it was the source of confusion in work today. I think the new change follows the guidelines for command examples:

> When in doubt, keep new command-line users in mind. Err on the side of clarity rather than terseness. For example, commands that require sudo should include it directly in the examples.

### CODE REVIEW

**Comment by andrewmarkallen** [comment]
Is it possible to merge this (pending approval of course)? I fixed my email address in a new commit but it's looking for the previous (misspelt, nonexistent email) to also sign CLA.

If not I have another branch ready to merge if you close this

**Comment by navarroaxel** [comment]
@andrewmarkallen we have an issue because the pages can have up to 8 example, and I count 9 here.

File: pages/common/conda.md
Comment (sbrl) [inline pages/common/conda.md:14]: Double space here?
```suggestion
- Load an environment:
```

**Review by sbrl — COMMENTED** [review]
Thanks for the PR, @andrewmarkallen! Unfortunately as @navarroaxel says we have a limit of 8 example per page, which means we're unable to merge until we drop below that limit.

However, in terms of grammar and syntax etc your edit looks good! Just 1 minor comment below.

Regarding your author issue with the commits: try squashing them together and force-pushing. In theory that should fix the problem?

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — CHANGES_REQUESTED** [review]
Thanks for the edit @navarroaxel, but as far as I can see we're still at 9 examples here.

**Comment by bl-ue** [comment]
@andrewmarkallen, thank you for this PR, but to me the new change doesn't improve clarity much.  A user will automatically associate "load" with "activate" and "unload" with "deactivate" when they see
```
- Load or unload an environment:

`conda {{activate|deactivate}} {{environment_name}}`
```
and this page already has 8 examples, which is the maximum.

**Comment by bl-ue** [comment]
You could open a PR to update that Python version in the 6th example--3.4 is really old.  3.9 is the current version...

**Comment by sbrl** [comment]
This is an old PR so it doesn't matter much, but next time @bl-ue try to talk the user through it, and give them an opportunity to close the PR themselves :-)

**Comment by bl-ue** [comment]
Okay @sbrl.  Maybe I was a little too eager to get issues/PR closed...

**Comment by sbrl** [comment]
No worries! It's a balance between closing old PRs and giving users time to act of feedback. Generally a month of inactivity is enough to close a PR, hence the stale bot. But a number of issues are long-term tracking issues, which makes them a lot more difficult to close.

**Comment by bl-ue** [comment]
Yeah, I know.  I'm working on trying to trim some of those down.  I've actually got 15+ closed since you added me as a collaborator 4 days ago.

**Comment by sbrl** [comment]
Wow, that's a lot @bl-ue!

**Pull request closed without merging**
